### ISSUE TITLE
Haxefilxel Demo

### ISSUE THREAD

**Issue by p0008874** [ruffle-rs/ruffle#11513]
### Describe the bug

Filters:
https://github.com/ruffle-rs/ruffle/assets/75534590/8ddb2592-a5ff-433c-b4c9-424946cbdd08
FlxSpriteFilters:
![ruffle_yj9OGir2cJ](https://github.com/ruffle-rs/ruffle/assets/75534590/3b513d45-c642-401e-af6e-1d981959a3d8)
FileBrowse:
```
2023-06-13T11:47:09.566167Z  INFO ruffle_core::player: Loaded SWF version 21, resolution 640x480 @ 60 FPS
2023-06-13T11:47:09.663721Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.utils.Dictionary constructor with weak keys
2023-06-13T11:47:09.731684Z  WARN run_frame:run_all_phases_avm2: ruffle_desktop::backends::storage: Unable to read file Os { code: 3, kind: NotFound, message: "系統找不到指定的路徑。" }
2023-06-13T11:47:09.732215Z  WARN run_frame:run_all_phases_avm2: ruffle_desktop::backends::storage: Unable to read file Os { code: 3, kind: NotFound, message: "系統找不到指定的路徑。" }
2023-06-13T11:47:09.739447Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.media.Sound constructor with arguments
2023-06-13T11:47:13.421273Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.ui.MouseCursorData.hotSpot setter
2023-06-13T11:47:13.421591Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.ui.MouseCursorData.data setter
2023-06-13T11:47:13.422629Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.ui.Mouse.registerCursor()
```

### Expected behavior

Filters:
https://github.com/ruffle-rs/ruffle/assets/75534590/ecd42ba9-5f2e-4254-aa64-b5d6b70158d7
FlxSpriteFilters:
![flashplayer_sa_debug_IaMLSd56kX](https://github.com/ruffle-rs/ruffle/assets/75534590/6c6b2217-57fe-48da-a4e9-4b9b67326034)
FileBrowse:
https://github.com/ruffle-rs/ruffle/assets/75534590/ea88e7ec-6348-4ce1-8528-ec22d8f9934e

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

https://demos.haxeflixel.com/swf/Filters.swf
https://demos.haxeflixel.com/swf/FlxSpriteFilters.swf
https://demos.haxeflixel.com/swf/FileBrowse.swf

**Comment by Briellenoel123** [comment]
If this merged, Glow and Drop Shadow will be implemented by #11878

**Comment by Lord-McSweeney** [comment]
Bevel and Displacement Map are the only two left.

**Comment by waspennator** [comment]
Bevel filter should come with #12018

**Comment by waspennator** [comment]
Bevel filter has been merged in, displacementmap and issue should be resolved by #12149

**Comment by Lord-McSweeney** [comment]
This looks almost exactly the same as Flash now. The only difference remaining now is that the "Displacement"/"Bevel" and the "(flash only)" text are vertically closer in Ruffle than they are in FP.
